libmultipath/nvme/linux/nvme.h

```diff
@@ -1035,7 +1035,7 @@ enum {
 	NVME_SANITIZE_LOG_STATUS_MASK		= 0x0007,
 	NVME_SANITIZE_LOG_NEVER_SANITIZED	= 0x0000,
 	NVME_SANITIZE_LOG_COMPLETED_SUCCESS	= 0x0001,
-	NVME_SANITIZE_LOG_IN_PROGESS		= 0x0002,
+	NVME_SANITIZE_LOG_IN_PROGRESS		= 0x0002,
```

Comment:
It's possible this comes from an upstream and thus can't be changed here.

Reply:
It comes from an upstream, but we can change it nonetheless. The symbol isn't used in our code. Some day we should switch to git submodules.